### ISSUE TITLE
BERTてなんなん？

### ISSUE THREAD

**Issue by 432ats** [432ats/scratch_NLP#2]
### BERT
2018年にGoogleが発表した自然言語処理の深層学習の手法
文章分類、翻訳、質問応答などのタスクで当時の最高スコアをマーク←「人間を超えた」なんて言われたり言われなかったり
Bidirectional Encoder Representations from Transformersの略
日本語で「Transformerによる双方向のエンコード表現」
これだけじゃよくわからない:confused:
- Transformerて何？ →　#1 
- 双方向てどういう意味？

ヒント
- pre-train（事前学習）
- fine-tuning（転移学習）

#### MLM
Masked Language Model(= MLM)

#### NSP
Next Sentence Prediction(= NSP)

### 今後どうなる
